dev/com.ibm.ws.security.wim.adapter.ldap_fat/fat/src/com/ibm/ws/security/wim/adapter/ldap/fat/FATTestIDSwithSSLTrustOnly.java

```diff
@@ -75,14 +130,13 @@ public static void setUp() throws Exception {
         }
     }
 
-    @AfterClass
-    public static void tearDown() throws Exception {
-        Log.info(c, "tearDown", "Stopping the server...");
-        try {
-            server.stopServer("CWIML4529E", "CWIMK0004E", "CWPKI0041W");
-        } finally {
-            server.deleteFileFromLibertyInstallRoot("lib/features/internalfeatures/securitylibertyinternals-1.0.mf");
-        }
+    /**
+     * Configure the embedded LDAP server.
+     *
+     * @throws Exception If the server failed to start for some reason.
+     */
+    private static void setupLdapServer() throws Exception {
+        ldapServer = new InMemoryTDSLDAPServer();
```

Comment:
If all we are doing is newing up an LDAP server that is already configured, you might just want to do that directly instead of wrapping it in this method.